src/lib/storage/chunk.hpp

```diff
@@ -151,6 +152,19 @@ class Chunk : private Noncopyable {
    */
   size_t estimate_memory_usage() const;
 
+  void increase_invalid_row_count(uint64_t count) const;
+
+  uint64_t invalid_row_count() const { return _invalid_row_count; }
+
+  /**
+   * Returns the commit-id of the MvccDeletePlugin-logical-delete-transaction
+   * in case it has been carried out.
```

Comment:
Please be more precise that "has been carried out". Since this is one of the main header files, it wouldn't hurt to explain the concept a bit more.